.coveragerc

```diff
@@ -12,3 +12,4 @@ exclude_lines =
     # Don't complain about debug code
     if Image.DEBUG:
     if DEBUG:
+    if DEBUG > 1:
```

Comment:
How does this relate to Python 2/3?

Reply:
It doesn't, it's just a single line, but shall I put it in its own PR?

---

In fact, it's only used in this one place:

https://github.com/python-pillow/Pillow/blob/211a1daa95634be89641b73956fb4e3ef11facff/src/PIL/TiffImagePlugin.py#L857

Where:

https://github.com/python-pillow/Pillow/blob/211a1daa95634be89641b73956fb4e3ef11facff/src/PIL/TiffImagePlugin.py#L70

And the other uses are like:

https://github.com/python-pillow/Pillow/blob/211a1daa95634be89641b73956fb4e3ef11facff/src/PIL/TiffImagePlugin.py#L751

Should we do this instead (and here or another PR)?
```diff
-            if DEBUG > 1:
+            if DEBUG:
```

Reply:
I've created PR #4112

Reply:
Thanks, I've rebased and dropped this commit.